toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -226,6 +226,67 @@ So, any existing ExtraCommandLine value in the base image will be replaced.
 If the partitions are not customized, then the `ExtraCommandLine` value will be appended
 to the existing `grub.cfg` file.
 
+### SELinux
+
+Specifies the mode to set SELinux to.
+
+If this field is not specified, then the existing SELinux mode in the base image is
+maintained.
+
+If SELinux is enabled, then all the file-systems that support SELinux will be have their
+file labels updated/reset (using the `setfiles` command).
+
+Supported options:
+
+- `disabled`: Disables SELinux.
+
+- `permissive`: Enables SELinux but only logs access rule violations.
+
+- `enforcing`: Enables SELinux and enforces all the access rules.
+
+- `force-enforcing`: Enables SELinux and sets it to enforcing in the kernel
+  command-line.
+  This means that SELinux can't be set to `permissive` using the `/etc/selinux/config`
+  file.
+  
+Example:
+
+```yaml
+SystemConfig:
+  KernelCommandLine:
+    SELinux: enforcing
+
+  PackagesInstall:
+  - selinux-policy
+  - selinux-policy-modules
+  - setools-console
+  - policycoreutils-python-utils
+```
+
+### SELinuxPolicy
+
+When SELinux is enabled, the Mariner Image Customizer tool ensures the image has the
+`selinux-policy` package installed.
+This package contains the default SELinux rules and is required for SELinux enabled
+images to be functional.
+
+However, some images may wish to use their own custom SELinux policy package.
```

Comment:
I would recommend rephrasing such that the functionality is described first and reasoning explained later. As a user, when looking at docs I'd want to be able to read through as quickly as possible to get the info I need.
Eg: This property allows the user to override the name of the policy package... (followed by the explanation / reasoning)

Reply:
Fixed.